tests/test_advection.py

```diff
@@ -1,15 +1,33 @@
-from parcels import (FieldSet, Field, ScipyParticle, JITParticle, ErrorCode, StateCode,
-                     AdvectionEE, AdvectionRK4, AdvectionRK45, AdvectionRK4_3D,
-                     AdvectionAnalytical, AdvectionDiffusionM1, AdvectionDiffusionEM)
-from parcels import ParticleSetSOA, ParticleFileSOA, KernelSOA  # noqa
-from parcels import ParticleSetAOS, ParticleFileAOS, KernelAOS  # noqa
-import numpy as np
-import pytest
 import math
 from datetime import timedelta as delta
-from parcels import logger
+
+import numpy as np
+import pytest
 import xarray as xr
 
+from parcels import (  # noqa
+    AdvectionAnalytical,
+    AdvectionDiffusionEM,
+    AdvectionDiffusionM1,
+    AdvectionEE,
+    AdvectionRK4,
+    AdvectionRK4_3D,
+    AdvectionRK45,
+    ErrorCode,
+    Field,
+    FieldSet,
+    JITParticle,
+    KernelAOS,
+    KernelSOA,
+    ParticleFileAOS,
+    ParticleFileSOA,
+    ParticleSetAOS,
+    ParticleSetSOA,
+    ScipyParticle,
+    StateCode,
+    logger,
+)
```

Comment:
Do you know why there's only one class per line in this import call (and similar files)? From the isort documentation, it seems that multiple classes are also supported. Or is this intentional because it's even cleaner?

Reply:
This is the way imports are formatted using the `black` auto formatter, which `isort` is compatible with by specifying the profile to be `black` (see the modified `setup.cfg`).

Although we don't use `black` at the moment, it is a really popular formatter that I think would be good to consider introducing in future. This is just a small step in that direction, preventing reformatting the imports again should we start using `black`.